client/service/src/task_manager/mod.rs

```diff
@@ -336,9 +336,12 @@ impl TaskManager {
 		}
 	}
 
-	/// Set what the task manager should keep alive.
-	pub(super) fn keep_alive<T: 'static + Send + Sync>(&mut self, to_keep_alive: T) {
-		self.keep_alive = Box::new(to_keep_alive);
```

Comment:
@expenses I was sure that you did make a tuple magic thingy here that would make this function callable multiple times :thinking: why was is it gone? I'm missing something...

Reply:
You mean `keep_alive((SOmehing, else, yolo))`

Reply:
I totally mean S0mething, else, yolo.

I think she refactored her code before pushing and since keep_alive here is privat-ish she didn't foresee it could be misused.

Clearly something to fix, it's a pitfall.

Reply:
Yeah SOmething :P You should change your font to distinguish O and 0 :P

Reply:
The mix between the monospace font and sans font tricked my eyes